README.md

```diff
@@ -52,6 +52,73 @@ following fields:
 * `events`: An event sink which supports the same interface as Node's `EventEmitter`. Required. See
   below for the events that get generated.
 
+### Emitting metadata about the document
+
+Every time the validator finds/deduces a piece of metadata about the document, it emits a `metadata` event.
+`metadata` messages contain two arguments: *key* and *value*.
+Keys are unique ID's, while the types of values are different according to the specific kind of metadata.
+
+These properties are now returned when found:
+
+* `docDate`: The date associated to the document.
+* `title`: The (possible) title of the document.
```

Comment:
Nothing, really. Just stressing that this one (like others) is an approximate one. The title returned could be the one found inside `<head><title>` or inside `<div class="head"><h1>`&hellip;